keyboards/wuque/ikki68_aurora/ikki68_aurora.h

```diff
@@ -0,0 +1,215 @@
+/* Copyright 2021 wuquestudio
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+
+/* Standard Arrangement
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐ ┌───┬───┐
+ * │00 │01 │02 │03 │04 │05 │06 │07 │08 │09 │0a │0b │0c │0d │2d │ │0e │0f │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┤ ├───┼───┤
+ * │10   │11 │12 │13 │14 │15 │16 │17 │18 │19 │1a │1b │1c │1d   │ │1e │1f │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    | └───┴───┘
+ * │20    │21 │22 │23 │24 │25 │26 │27 │28 │29 │2a │2b │2c │    │
+ * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴┬───┤ ┌───┐
+ * │30  |31 │32 │33 │34 │35 │36 │37 │38 │39 │3a │3b │3c    |3d │ │3e │
+ * ├────┼───┴┬──┴─┬─┴───┴──┬┴───┼───┴───┴──┬┴───┼───┴┬────┬┴─┬─┴─┼───┼───┐
+ * │40  │41  │42  │44      |46  |48        │4a  │4b  │4c  │  │4d │4e │4f |
+ * └────┴────┴────┴────────┴────┴──────────┴────┴────┴────┘  └───┴───┴───┘
+ */
+#define LAYOUT_default( \
```

Comment:
This contains all key positions? 

If so, may be better to name it `LAYOUT_all`

Reply:
Yes, it contains all key positions, but it is also the firmware that is flashed into the keyboard by default.

Reply:
I think default is fine )

Reply:
For a layout with all possible positions, we highly recommend `LAYOUT_all`, so that it's explicitly clear.  `LAYOUT_default` has a very different implication from `LAYOUT_all`, in that it doesn't imply that every possible position is referenced, but rather that it's the default/recommended layout.

So, in this case, definitely please use `LAYOUT_all` instead.

Reply:
OK, I fixed this ) @drashna